maxim/logger/mistral/client.py

```
@@ -39,6 +39,21 @@ def _setup_logging(
                     "messages": MistralUtils.parse_message_param(messages),
                 }
             )
+            input = None
```

Pylint (3.3.7)

[warning] Redefining built-in 'input'

(W0622)